src/pages/update-record.js

```diff
@@ -74,8 +74,28 @@ const UpdateRecord = () => {
       reset(record[0]);
       selectRegion(record[0].region_id);
     }
+
+    Array.prototype.forEach.call(
+      document.querySelectorAll("input[type=text]"),
+      function (input) {
+        input.addEventListener("keyup", function () {
+          input.value = input.value.toUpperCase();
+        });
+      }
```

Comment:
Since now have the code in a useEffect below, how about removing this one here as it is no longer needed